packages/bsky/src/api/app/bsky/feed/getLikes.ts

```diff
@@ -37,18 +37,20 @@ export default function (server: Server, ctx: AppContext) {
       })
 
       const likesRes = await builder.execute()
-      const actors = await services.actor(db).views.profile(likesRes, requester)
+      const actors = await services
+        .actor(db)
+        .views.profiles(likesRes, requester)
 
       return {
         encoding: 'application/json',
         body: {
           uri,
           cid,
           cursor: keyset.packFromResult(likesRes),
-          likes: likesRes.map((row, i) => ({
+          likes: likesRes.map((row) => ({
             createdAt: row.createdAt,
             indexedAt: row.indexedAt,
-            actor: actors[i],
+            actor: actors[row.did],
           })),
```

Comment:
I don't think it's really a new issue here, but we may want to skip over likes that don't have a given `actors[row.did]` rather than return a potentially invalid response.